content/induction/sprints/1/prep/index.md

```diff
@@ -0,0 +1,358 @@
+
+Describe some of the challenges that Ahmed and Naima face when trying to write a blog together in this way.
+
+Create or join a thread 🧵 on Slack to share your answers. Did your cohort colleagues think of similar things?
+
+{{</note>}}
+
+## ⌛ Version control software
+
+> To improve their way of working, Ahmed and Naima realise they need the following:
+>
+> - a way to create and manage the different versions of their project
+> - a way to share the history of the project between different users
+
+To manage the different versions of the blog project, they decide to use **Git**.
+
+{{<note type="definition" title="definition: Git">}}
+
+Git is **version control software** that allows developers to create and manage different versions of a project.
+
+{{</note>}}
+
+In Git, we create different versions of a project over time by creating a **commit**.
+
+A **commit** is a snapshot of our project at a particular point in time. You can also think of a **commit** as a particular version of a project.
+
+**Commits** store the following information👍
+
+- what changed in this commit
+- who created the change
+- what time the change happen
+- what the previous commit was
+
+A typical timeline of commits might look like this:
+
+![commit-history](commit-history.png)
+
+## 🗄️ 🌐 Sharing history
+
+Earlier on, Ahmed and Naima realised they also need the following:
+
+> a way to share the history of the project between different users
+
+To share a project and its history, we can use an online platform called **Github**
+
+**Github** is a platform where teams can store projects along with a history of their different versions.
+
+By storing projects on Github, multiple users can gain access to the history of a project.
+
+{{<note type="definition" title="Definition: Repository">}}
+On Github we call our project and its history a **repository**.
+{{</note>}}
+
+{{<note type="exercise" title="Exercise 2.1">}}
+
+### Explore 🔍
+
+In this exercise, you'll need to explore a **Github repository.**
+
+You'll need to look around and figure out where to find different files and find out information about them.
+
+**⚠️ You won't be expected to know what the files do at this stage.**
+
+Go to the following link: https://github.com/CodeYourFuture/education-blog
+
+It will take you to a Github repository called **education-blog**.
+
+Answer the following questions using the page linked to above:
+
+a) View the **README.md** file. What do the instructions tell you?
+b) How many files are there inside the `blogs` folder?
+c) How many lines are there in the **package.json** file?
+d) Find the file with the blog content you can see on the live site here [blog 1](https://git-demo-week1.netlify.app/blogs/1)
+
+You'll learn more about these type of files throughout the course.
+
+{{</note>}}
+
+> We can use the Github interface to explore the different **commits** ( versions ) of a project too.
+
+{{<note type="exercise" title="exercise 2.2">}}
+
+### Explore 🔍
```

Comment:
I actually did have this section in the original draft and then took it out - I do agree I think it would be super useful for them to browse the repository somewhere in the past!
I'll reinstate that section.